hooks/hookexecution/outcome.go

```diff
@@ -77,14 +78,15 @@ type GroupOutcome struct {
 type HookOutcome struct {
 	// ExecutionTime is the execution time of a specific hook without applying its result.
 	ExecutionTime
-	AnalyticsTags hookanalytics.Analytics `json:"analytics_tags"`
-	HookID        HookID                  `json:"hook_id"`
-	Status        Status                  `json:"status"`
-	Action        Action                  `json:"action"`
-	Message       string                  `json:"message"` // arbitrary string value returned from hook execution
-	DebugMessages []string                `json:"debug_messages,omitempty"`
-	Errors        []string                `json:"-"`
-	Warnings      []string                `json:"-"`
+	AnalyticsTags hookanalytics.Analytics    `json:"analytics_tags"`
+	HookID        HookID                     `json:"hook_id"`
+	Status        Status                     `json:"status"`
+	Action        Action                     `json:"action"`
+	Message       string                     `json:"message"` // arbitrary string value returned from hook execution
+	DebugMessages []string                   `json:"debug_messages,omitempty"`
+	Errors        []string                   `json:"-"`
+	Warnings      []string                   `json:"-"`
+	SeatNonBid    openrtb_ext.NonBidsWrapper `json:"-"`
```

Comment:
@ashishshinde-pubm :  `SeatNonBid ` -  Is it pointer variable?

Reply:
No, its struct which contains map.
Idea is that each HookOutcome should just return SeatNonBid copy and caller should not modify it.